guilib/measure/uimeasurement.py

```diff
@@ -665,13 +683,14 @@ def __make_device(self, device_cls, device_name, **other_info):
         elif issubclass(device_cls, CameraABC):
             section = "camera_settings"
 
+        device_name = settings_info.more['name'] or settings_info.unique_name
         device.settings[section]['device_name'] = device_name
         new_cfg_path = Path(_cfg_dir) / f"{device.name_clean}.ini"
         if new_cfg_path.exists():
             print(f"{section} config file name conflict! Overwriting existing")  # TODO: ask what to do with the (invalid) file
         with new_cfg_path.open('w', encoding='utf-8') as fproxy:
             device.settings.write(fproxy)
-
+        device.uses_default_settings = False
```

Comment:
I now see why you made `.uses_default_settings` public. Discard my comments in `abc` concerning this.